redistributable/SpringBoot/generated.iml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file from your PR? I don't think this was intentional.

Reply:
removed